### ISSUE TITLE
ARROW-134 bson.errors.InvalidDocument: cannot encode object: <NA>, of type: <class 'pandas._libs.missing.NAType'>

### ISSUE THREAD

**Issue by etiennellipse** [mongodb-labs/mongo-arrow#117]
Package version: 0.6.3
PyMongo version: 4.3.2

Pretty exciting library for us. We use pandas DataFrame heavily, and store data to MongoDB.

I am inserting data from pandas dataframes to MongoDB. Many of them contain integer with nullable values, which are encoded to pandas [Nullable integer type](https://pandas.pydata.org/docs/user_guide/integer_na.html) which use the pandas.NA value to represent nulls (not numpy.nan).

Here is a simple reproduction of the issue:
```
import pandas as pd
from pymongoarrow.api import write

from src import db


db = db.get_college_db("qweqwe", "historic")

df = pd.DataFrame({
    "one": [1, 2, 3, None],
})
df["one"] = df["one"].astype("Int64")

print(df.dtypes)

write(db.collection, df)

```

Raises error:
`bson.errors.InvalidDocument: cannot encode object: <NA>, of type: <class 'pandas._libs.missing.NAType'>`

**Comment by etiennellipse** [comment]
I observed the same error with the pandas "string" type (new in pandas 1.0.0), which uses pd.NA too.

**Comment by blink1073** [comment]
Hi @etiennellipse, thank for you bringing this to our attention.  I've opened https://jira.mongodb.org/browse/ARROW-134 to track a fix.

**Comment by ShaneHarvey** [comment]
This was fixed in https://github.com/mongodb-labs/mongo-arrow/commit/60e77060409a6cdabc2d2a62e87e1e59f9ea41a2 and released in 0.7.0.
